cirq/google/calibration/phased_fsim.py

```diff
@@ -130,3 +140,217 @@ def override_by(self, other: 'PhasedFSimCharacterization') -> 'PhasedFSimCharact
             other that are not None). Otherwise the current values are used.
         """
         return other.merge_with(self)
+
+
+# TODO: Add start and end calibration timestamp
+# TODO: Add export to Panda's data frame
+@dataclasses.dataclass(frozen=True)
+class PhasedFSimCalibrationResult:
+    """The PhasedFSimGate characterization result.
+
+    Attributes:
+        parameters: Map from qubit pair to characterization result. For each pair of characterized
+            quibts a and b either only (a, b) or only (b, a) is present.
```

Comment:
should we just always sort the key, i.e. canonicalize the key?

edit: I see below, that you have to call `parameters_for_qubits_swapped` to get the other ordering. This doesn't obviate the utility of storing things with a canonical key

Reply:
Canonalization is not a requirement for QE Floquet calibration so I didn't enforce it here. This way it is more flexible if people would create requests on their own. I'm not particularly attached to that decision tough.

Reply:
ack